### PR TITLE
Create a template to support for Horizontal Pod Autoscaling (HPA)

### PR DESCRIPTION
**Pull request by gicappa**
Horizontal Pod Autoscaling support for the microservices `runtime-bundle` and `activiti-cloud-query`.

HPA allows to scale up or down the number of pods when the metrics (CPU and memory) collected on the workload cross a threshold.

A new configuration (similar to the following one) is possible:

```yaml
runtime-bundle:
  hpa:
    enabled: true
    minReplicas: 1
    maxReplicas: 6
    cpu: 90
    memory: "2000Mi"
activiti-cloud-query:
  hpa:
    enabled: true
    minReplicas: 1
    maxReplicas: 4
    cpu: 90
```

These values automatically scale the ReplicaSet when the average CPU consumption rises above 90% or the memory consumption overcomes 2GB.

Two scaleDown policies have been added by default to the HPA configuration to avoid scaling down too quickly.
```yaml
scaleDown:
    policies:
      - type: Pods
        value: 1
        periodSeconds: 60
      - type: Percent
        value: 15
        periodSeconds: 60
      selectPolicy: Max
```

Ref: [Activiti/Activiti#3791](https://github.com/Activiti/Activiti/issues/3791)

### CODE REVIEW

File: charts/common/tests/hpa_test.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+      - hasDocuments:
+          count: 0
+  - it: should render nothing if hpa.enabled=false
+    set:
+      enabled: true
+      hpa:
+        enabled: false
+    asserts:
+      - hasDocuments:
+          count: 0
```
Comment (erdemedeiros) [inline charts/common/tests/hpa_test.yaml:20]: This test is covering the same thing as the previous one.

File: charts/common/tests/hpa_test.yaml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  - it: should render minReplicas
```
Comment (erdemedeiros) [inline charts/common/tests/hpa_test.yaml:21]: Suggestion: `should render minReplicas and maxReplicas`.

**Review by erdemedeiros — CHANGES_REQUESTED** [review]
Some small details in the tests. LGTM otherwise!

File: charts/common/tests/hpa_test.yaml (hunk 1 above)
Comment (gicappa) [reply charts/common/tests/hpa_test.yaml]: Thanks, fixed.

File: charts/common/tests/hpa_test.yaml (hunk 2 above)
Comment (gicappa) [reply charts/common/tests/hpa_test.yaml]: Done.

**Review by igdianov — APPROVED** [review]
Lgtm

**Review by andrea-ligios — APPROVED** [review]
LGTM!

**Review by erdemedeiros — APPROVED** [review]
LGTM!

**Pull request merged**